tests/system/projects/test_project.py

```diff
@@ -288,6 +288,62 @@ def test_local_pipeline(self):
     def test_kfp_pipeline(self):
         self._test_new_pipeline("kfppipe", engine="kfp")
 
+    def _test_remote_pipeline_from_github(
+        self, name, workflow_name, engine=None, local=None, watch=False
+    ):
+        project_dir = f"{projects_dir}/{name}"
+        shutil.rmtree(project_dir, ignore_errors=True)
+        project = mlrun.load_project(
+            project_dir, "git://github.com/mlrun/project-demo.git", name=name
+        )
+        # Skipping until mlrun/project-demo will contain newflow workflow in project
+        workflow_names = [workflow["name"] for workflow in project.spec.workflows]
+        if workflow_name not in workflow_names:
+            self._delete_test_project()
+            return
+        workflow_dict = project.spec._workflows[workflow_name].to_dict()
+        run = project.run(
+            workflow_name,
+            watch=watch,
+            local=local,
+            engine=engine,
+        )
+
+        assert run.state == mlrun.run.RunStatuses.succeeded, "pipeline failed"
+        assert run.run_id, "workflow's run id failed to fetch"
+        assert (
+            workflow_dict.items() <= run.workflow.to_dict().items()
+        ), "workflow executed with wrong parameters"
+        self._delete_test_project()
```

Comment:
is this required in your other tests as well?

Reply:
removed.. not really required